packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -18,10 +18,12 @@ type ActionTriggerBase = {
 };
 
 type ActionTriggerWithButton = ActionTriggerBase & {
+    type: "button";
     onClick: () => void;
 };
 
 type ActionTriggerWithLink = ActionTriggerBase & {
+    type: "link";
     href: string;
     onClick?: () => void;
 };
```

Comment:
I bumped `wonder-blocks-banner` by a "major" version because this is a breaking change.